### DISCUSSION TITLE
如何自定义 quick open 的高度

### DISCUSSION THREAD

**Discussion by PerfectPan** [opensumi/core#2572] [Q&A]
看代码是写死了高度 https://github.com/opensumi/core/blob/main/packages/quick-open/src/browser/quick-open.service.tsx#L90，
实际表现：
![image](https://user-images.githubusercontent.com/24316656/231106626-95ef233f-753c-43ea-8efa-5dc2f289e235.png)
想自定义高度实现类似 VSCode 上渲染在 tabbar 下面的效果，即：
![image](https://user-images.githubusercontent.com/24316656/231106838-01d464e7-430d-4e36-a87a-98de759cdf17.png)
目前是写了个 Module 去改 container 的高度，想问问是不是有别的方法：
```ts
import { Injectable, Autowired } from '@opensumi/di';
import { BrowserModule, ClientAppContribution, Domain } from '@opensumi/ide-core-browser';
import { QuickOpenService } from '@opensumi/ide-quick-open';

@Domain(ClientAppContribution)
class QuickOpenContribution implements ClientAppContribution {
  @Autowired(QuickOpenService)
  private quickOpenService: QuickOpenService;

  onDidStart() {
    this.quickOpenService.hide();
    const quickOpenContainer = document.getElementsByTagName('quick-open-container')[0];
    if (quickOpenContainer) {
      (quickOpenContainer as any).style.top = '82px';
    }
  }
}


@Injectable()
export class QuickOpenPatchModule extends BrowserModule {
  providers = [QuickOpenContribution];
}
```

**Comment by erha19** [comment]
@hacke2 看一下这个问题？感觉可以支持一个配置用于控制 QuickOpen 的顶部高度，参考 `debug.toolbar.top` 参数（用于控制调试面板 Top 数值）
